Zend/zend_errors.h

```diff
@@ -37,10 +37,12 @@
 #define E_RECOVERABLE_ERROR	(1<<12L)
 #define E_DEPRECATED		(1<<13L)
 #define E_USER_DEPRECATED	(1<<14L)
+#define E_DEBUG            (1<<15L)
+#define E_USER_DEBUG       (1<<16L)
 
-#define E_EXCEPTION			(1<<15L)
+#define E_EXCEPTION			(1<<17L)
```

Comment:
@Danack  this is irrelevant … E_EXCEPTION is PHP 7 only and never exported as constant to userland.